### PR TITLE
[Feature] Add Dockerfile

### PR DESCRIPTION
**Pull request by Ezra-Yu**
1. Add Dockfile, the builded image can be attached in the dockerhub.

### CODE REVIEW

**Comment by mzr1996** [comment]
So how to use the dockerfile ?

**Comment by Ezra-Yu** [comment]
> So how to use the dockerfile ?

Install docker, and then  use the Dockerfile to build mmcls image

```bash
docker build -t mmcls:${Image_Name}
 ```
this web [docker-gitbook](https://yeasy.gitbook.io/docker_practice/) show you how to install and use docker.

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+RUN apt-get update && apt-get install -y ffmpeg libsm6 libxext6 git ninja-build libglib2.0-0 libsm6 libxrender-dev libxext6 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install MMCV
+RUN pip install mmcv-full==1.3.8 -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.6.0/index.html
+
+# Install MMClassification
+RUN conda clean --all
+RUN git clone https://github.com/open-mmlab/mmclassification.git
```
Comment (mzr1996) [inline docker/Dockerfile:20]: Should we use an absolute path? 
Refers to https://github.com/open-mmlab/mmaction2/blob/aa26715d5c6a0991af4ead5b5ea03c46dd65dcee/docker/Dockerfile#L20-L21.

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+WORKDIR ./mmclassification
+RUN pip install --no-cache-dir -e .
```
Comment (mzr1996) [inline docker/Dockerfile:22]: Is a data folder necessary?
Refers to https://github.com/open-mmlab/mmaction2/blob/aa26715d5c6a0991af4ead5b5ea03c46dd65dcee/docker/Dockerfile#L22
I tested on my computer and it's not necessary, but I cannot confirm it with an old version of docker.

Comment (Ezra-Yu) [reply docker/Dockerfile:22]: Unnecessary，if the user need a "data" dir, he can mkdir by himself in the container.

**Review by mzr1996 — APPROVED** [review]
LGTM

**Pull request merged**